xml/dm_registers.xml

```diff
@@ -140,8 +147,14 @@
 
             Writes apply to the new value of \Fhartsel and \Fhasel.
         </field>
-        <field name="0" bits="28:27" reset="0" access="R" />
-         <field name="hasel" bits="26" access="R/W" reset = "0">
+	<field name="haveresetack" bits="28" access="R/W" reset="0">
```

Comment:
"ackreset"? "haveresetack" sounds like a bit that indicates whether "reset ack" is supported or not.

Reply:
how about 'ackhavereset'? I do like tying the bits together since there are a lot of resets one could be acking

Reply:
Perfect!